parsons/zoom/zoom.py

```diff
@@ -158,7 +160,15 @@ def get_users(self, status="active", role_id=None):
         logger.info(f"Retrieved {tbl.num_rows} users.")
         return tbl
 
-    def get_meetings(self, user_id, meeting_type="scheduled"):
+    def get_meetings(
+        self,
+        user_id,
+        meeting_type: Literal[
+            "scheduled", "live", "upcoming", "upcoming_meetings", "previous_meetings"
+        ] = "scheduled",
+        from_date: Optional[datetime.date] = None,
```

Comment:
Can you add the two new input params, from_date and to_date, and the fact that they're datetime objects. to the docstring?

Reply:
Done with 9d988c9991af83a923951448bc602cb0be7161ef